### PR TITLE
Update done comment and fix marking issue as "Done"

### PR DESCRIPTION
**Pull request by slide**
This updates the "Done" comment to add more information for the person (must access github invitation, issue location, etc) as well as changing to getting the transition ID programatically using the API.

### CODE REVIEW

**Comment by slide** [comment]
@oleg-nenashev Please review

File: src/main/java/org/jenkinsci/backend/ircbot/JiraHelper.java [hunk 1]
```diff
@@ -138,4 +136,18 @@ static String getFieldValueOrDefault(@Nonnull Issue issue, @Nonnull String field
         }
         return res;
     }
+
+    static Iterable<Transition> getTransitions(@Nonnull Issue issue) throws IOException, ExecutionException, TimeoutException, InterruptedException {
+        JiraRestClient client = createJiraClient();
+        return client.getIssueClient().getTransitions(issue).get(IrcBotConfig.JIRA_TIMEOUT_SEC, TimeUnit.SECONDS);
+    }
+
+    static Transition getTransitionByName(@Nonnull Iterable<Transition> transitions, String name) {
```
Comment (oleg-nenashev) [inline src/main/java/org/jenkinsci/backend/ircbot/JiraHelper.java]: CheckForNull please

File: src/main/java/org/jenkinsci/backend/ircbot/JiraHelper.java [hunk 2]
```diff
@@ -138,4 +136,18 @@ static String getFieldValueOrDefault(@Nonnull Issue issue, @Nonnull String field
...
+        for (Transition transition : transitions) {
+            if (transition.getName().equals(name)) {
```
Comment (oleg-nenashev) [inline src/main/java/org/jenkinsci/backend/ircbot/JiraHelper.java]: NIT: Maybe needs to be case-insensitive. Likely "no"

**Comment by oleg-nenashev** [comment]
Looks good to me. Ideally needs some testing. Assigning to myself

**Comment by oleg-nenashev** [comment]
As we discussed with @slide several days ago, for me the fix fails with the same error. Even after the bot's permissions fix on Jira

**Pull request closed without merging**